webapp/tests/test_functions.py

```diff
@@ -2053,6 +2053,7 @@ def test_sortByMinima(self):
         result = functions.sortByMinima({}, seriesList)
         self.assertEqual(result, expectedResult)
 
+    @patch('graphite.render.evaluator.prefetchData', lambda *_: None)
```

Comment:
This patch is needed to avoid the real `prefetchData` being called when patching `fetchData` in the test.